website/src/controllers/auth.js

```diff
@@ -65,10 +68,43 @@ api.authWithUrl = function(req, res, next) {
   });
 }
 
+var SECRET = "6LfGQ8wSAAAAADxUZbqjnRpMVIUJ9McpzvZt92iV";
```

Comment:
This also should be a config variable. Note: if you plan on using your recaptcha account for a future project, you will want to regenerate the secret key.